src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CATEGORY_IN_FILE_SUPPLY = "supply";
```

Comment:
Let's this about more concise naming (e.g. SUPPLY_OPERATION), also check other constant names please.